libraries/chain/db_notify.cpp

```diff
@@ -259,7 +259,6 @@ struct get_impacted_account_visitor
    void operator()( const custom_authority_create_operation& op )
    {
       _impacted.insert( op.fee_payer() ); // account
-      add_authority_accounts( _impacted, op.auth );
```

Comment:
Where was this moved to?

Reply:
I've just removed it because field op.auth was removed.

Reply:
What is the replacement for this logic?

Reply:
How is the status here?

Reply:
This will be ready tomorrow. I've faced some issues during implementation.

Reply:
Done